src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetTransferFinancialBreakdownRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests
+{
+    public class GetTransferFinancialBreakdownRequest : IGetApiRequest
+    {
+        public long AccountId { get; }
+
+        public GetTransferFinancialBreakdownRequest(long accountId)
+        {
+            AccountId = accountId;
+        }
+
+        public string GetUrl => $"accounts/{AccountId}/accountprojection/detail?numberOfMonths=12&startDate={DateTime.Now.Year}-04-06";
```

Comment:
I merged this code into this branch, 
I have created an extension method for the DateTime. 
This will return a Year based on this logic

return (dateTime.Month >= 4 ? dateTime.AddYears(1).ToString("yyyy") : dateTime.ToString("yyyy"));

I can use this like this
**startDate={DateTimeExtensions.ToFinancialYear(DateTime.Now)}-04-06";**

Reply:
So the financial year will be 2022 up till 1st April and then it will be 2023

Reply:
Elsewhere we have extension methods that give, say, the first day of the financial year - you could implement the same, and then just take the year component. This would be more reusable than an extension method that just returns the year as a string imo.